scripts/create-release-branch.js

```diff
@@ -5,30 +5,17 @@ const { checkVSCodeVersion, checkBaseBranch } = require('./validation-utils');
 const logger = require('./logger-util');
 const changeLogGeneratorUtils = require('./change-log-generator-utils');
 
+const RELEASE_TYPE = process.env['RELEASE_TYPE'];
+
 shell.set('-e');
 shell.set('+v');
 
-function getReleaseType() {
```

Comment:
This wasn't necessary since it was only used when getting the release type from the VS Code task. This is all done in GHA now, so we can get at the release type directly at line 8.